components/src/lists/TitledList.tsx

```diff
@@ -1,53 +1,53 @@
-// @flow
 // TitledList component
 import * as React from 'react'
 import cx from 'classnames'
 
 import styles from './lists.css'
-import { type IconName, Icon } from '../icons'
+import { Icon } from '../icons'
+import type { IconName } from '../icons'
 
-export type TitledListProps = {|
+export interface TitledListProps {
   /** text of title */
-  title: string,
+  title: string
   /** optional icon left of the title */
-  iconName?: ?IconName,
+  iconName?: IconName | null
   /** props passed down to icon (`className` and `name` are ignored) */
-  iconProps?: $Diff<React.ElementProps<typeof Icon>, { name: mixed }>,
+  iconProps?: Omit<React.ComponentProps<typeof Icon>, 'name'>
   /** optional data test id for the container */
-  'data-test'?: string,
+  'data-test'?: string
   // TODO(mc, 2018-01-25): enforce <li> children requirement with flow
   /** children must all be `<li>` */
-  children?: React.Node,
+  children?: React.ReactNode
   /** additional classnames */
-  className?: string,
+  className?: string
   /** component with descriptive text about the list */
-  description?: React.Node,
+  description?: JSX.Element
```

Comment:
Should be `React.ReactNode`

```suggestion
  description?: React.ReactNode
```